src/NuGet.Core/NuGet.Packaging/ContentModel/Infrastructure/Parser.cs

```diff
@@ -206,7 +206,7 @@ public TokenSegment(string token, char delimiter, bool matchOnly, PatternTable t
                             }
                             if (StringComparer.Ordinal.Equals(_token, "tfm"))
                             {
-                                item.Properties.Add("tfm_raw", substring);
+                                item.Properties.Add("tfm_raw", substring.ToString());
```

Comment:
Outside of this PR, but probably perfect case to have this pulled from a name table so we only ever create one of these strings per "raw TFM".

Reply:
👍